css/style.css

```diff
@@ -16,4 +16,137 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.navbar {
+    background-color: white;
+    position: fixed;
+    width: 100%;
+    z-index: 999; /*only used with position-999 is to keep it always as top layer*/
+    margin: 0;
+    padding: 0;
+    border: 0;
+    display: flex;
+    justify-content: space-around;
+}
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+    display: flex;
+}
+
+.logo {
+    height: 50px;
+    margin: 10px;
+    padding: 0;
+    border: 0;
+    width: 26;
+}
+
+li{
+    list-style-type: none;
+    display: inline-block;
+    margin: 25px 0 0 25px;
+    cursor: pointer;
+    justify-content: space-between;
+}
+
+.links {
+    float: right;
+    margin: 0,30;
+    padding: 0;
+    border: 0;
+    display: flex;
+    flex-direction: row;    
+}
+
+.introduction {
+    background-image: url('../img/first-background.jpg');
+    background-position: center-bottom;
+    background-size: cover;
+    height: 800px;
+    background-color: maroon;
+    display: flex;
+    flex-direction: column;
+}
+
+.introcontent {
+    color: white;
+    text-align: center;
+    font-weight: 300;
+    margin: 350px;
+    display: inline-block;
+    flex-direction: column;
+}
+
+h2 {
+    font-weight: 400;
+    font-size: 144;
+    margin-top: 2rem;
+    margin-right: 0px;
+    margin-bottom: 3.125rem;
+    margin-left: 0px;
+}
+
+.button {
+    width: 155px;
+    background-color: red;
+    border-color: red;
+    border-radius: 4px;
+    padding: 10px 6px;
+    font-size: 16px;
+    font-weight: bold;
+    text-align: center;
+    cursor: pointer;
+    border-width: 2px;
+}
+
+.cases {
+    display: flex;
+    flex-direction: column;
+    margin-top: 500;
+    text-align: center;
+}
+
+.devices {
+    background-color: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 0 300px 200px 300px;
+}
+
+.captions {
+    background-color: white;
+    display: inline-block;
+    flex-direction: column;
+    justify-content: space-around;
```

Comment:
as display has been set to `inline-block` so these two flex properties won't quite work